translations/base-it.yaml

```diff
@@ -317,47 +319,46 @@ ingame:
             itemsPerSecond: <x> oggetti / s
             itemsPerSecondDouble: (x2)
 
-            tiles: <x> titles
+            tiles: <x> caselle
 
     # The notification when completing a level
     levelCompleteNotification:
         # <level> is replaced by the actual level, so this gets 'Level 03' for example.
         levelTitle: Livello <level>
         completed: Completato
-        unlockText: Sbloccato <reward>!
+        unlockText: Sbloccato: <reward>!
```

Comment:
Change it to two lines and 'tab' the second one:
`unlockText: >-`
`    Sbloccato: <reward>!`